docs/source/tips-gcp.rst

```diff
@@ -49,3 +49,44 @@ If you are working on Debian or Ubuntu Linux distribution and have ``root`` perm
    sudo apt-get -yqm update
    sudo apt-get install -yq kubectl python3-distutils
 
+
+.. _gcp_free_trial:
+
+Using the Free Trial at GCP
+--------------------------
+
+GCP has a Free Trial for new users (https://cloud.google.com/free).  The Free Trial comes with some restrictions that are important for ElasticBLAST users.  These include only being able to run eight core concurrently and limiting the persistent disk size to 250G (https://cloud.google.com/terms/free-trial).  Normally, ElasticBLAST would run more than eight cores at a time and the default persistent disk size is 3000G.  
+
+You should be able to run ElasticBLAST under the Free Trial following the instructions at :ref:`quickstart-gcp`, but you will need to modify the configuration file to use fewer resources. You may not be able to use the cloud shell and the instance suggested below as that may exceed the quota on cores allowed at one time.  In that case, you will need to submit your ElasticBLAST search from your own computer.
+
+Remember also that the Free Trial only lasts 90 days.
```

Comment:
This may change again, so I'd recommend referring to GCP for details.
```suggestion
Fro additional details about GCP's free tier (duration, products included, etc), please visit https://cloud.google.com/free/docs/gcp-free-tier .
```

Reply:
The free tier allows one e2-micro (0.25 CPU and 1GB memory) machine per month. One cannot run even a tiny search on it. I am not sure if it is worth directing users there.

Reply:
GCP Free Tier has the miniscule machines etc that can be used all the time.  The Free Trial allows new users to run up to $300 worth of work on smallish (but not crazy small) machines.  See https://cloud.google.com/free on left column.